Dockerfile

```diff
@@ -1,10 +1,9 @@
 
 # --platform=$BUILDPLATFORM is used build javascript source with host arch
 # Otherwise TS builds on emulated archs and can be extremely slow (+1h)
-FROM --platform=${BUILDPLATFORM:-amd64} node:22-alpine as build_src
+FROM --platform=${BUILDPLATFORM:-amd64} node:22 as build_src
 ARG COMMIT
 WORKDIR /usr/app
-RUN apk update && apk add --no-cache g++ make python3 py3-setuptools && rm -rf /var/cache/apk/*
```

Comment:
> using regular node images for build layers; this remove the need to add extra tooling via apk every run

this layer will be retrieved from cache after the first run, but rather than reducing build time, a good argument for using the `node:22` full image as base is that we reduce the likelyhood that systems deps are missing.

But overall, this might increase the build time in the CI as the `node:22` image is not cached there, and afaik quite large to download.

Reply:
I would expect that if download time is an issue for CI, there would be ways to improve this via some caching.
The main goal of those changes is to speedup docker builds during dev cycles.

Reply:
> The main goal of those changes is to speedup docker builds during dev cycles.

it's not clear to me then why we need to use a full `node:22` as based on my observations the apk install layer will always be retrieved from cache after first run

Reply:
Hmm can't reproduce what I believe I was seeing during the interop indeed.

Reply:
did some testings with offline mirror vs. without, the build time seems to be roughly the same. The reason I would assume for this is that I have anyways all packages cached so there shouldn't be a difference.

> Hmm can't reproduce what I believe I was seeing during the interop indeed.

maybe it still does some checks against the yarn registry? There were definitely request timeouts I noticed during the interop which cwe might be able to avoid by using a offline mirror.